custom-connectors/HR Cloud/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file is completely empty. Please follow this sample to update: https://github.com/microsoft/PowerPlatformConnectors/tree/dev/certified-connectors/AgilePoint%20NX

Reply:
I forgot to upload changes to this file. Thanks a lot for the info!